<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeSlicerWindow</class>
 <widget class="QWidget" name="VolumeSlicerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VolumeSlicer</string>
  </property>
  <widget class="QSlider" name="xSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="zSlider">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="ySlider">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="decriptionLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Use the sliders to slice through the volume in the X, Y, and Z directions</string>
   </property>
  </widget>
  <widget class="QLabel" name="xSlice">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="ySlice">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="zSlice">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="xSliceImage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>X SLICE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ySliceImage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Y SLICE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="zSliceImage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Z SLICE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="xProjectionImage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>X PROJECTION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="yProjectionImage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Y PROJECTION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="zProjectionImage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Z PROJECTION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>660</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Projections</string>
   </property>
  </widget>
  <widget class="QLabel" name="zProj">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="yProj">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y Axis</string>
   </property>
  </widget>
  <widget class="QLabel" name="xProj">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X Axis</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
